dashboard/test/factories/factories.rb

```diff
@@ -797,6 +797,16 @@
     script_level
   end
 
+  factory :lesson_activity do
+    sequence(:position)
+    lesson
+  end
+
+  factory :activity_section do
+    sequence(:position)
+    activity
```

Comment:
should this say lesson_activity?

Reply:
`activity` was correct when that was the association name. factorybot knows from the association definition that `activity` uses to the LessonActivity model. but now that I've renamed the association per your other PR feedback, `lesson_activity` is correct here. (also unit tests will fail if this does not match the association name)